packages/components/src/components/ContentfulContainer/ContentfulContainer.tsx

```diff
@@ -38,10 +38,18 @@ export const ContentfulContainer: React.FC<ContentfulContainerAsHyperlinkProps>
   const { renderDropzone, node } = sectionProps;
   // Extract properties that are only available in editor mode
 
-  return renderDropzone(node, {
-    ['data-test-id']: 'contentful-container',
-    id: 'ContentfulContainer',
-    className: combineClasses(className, 'defaultStyles'),
-    WrapperComponent: Flex,
-  });
+  const isEmpty = !node.children.length;
+
+  return (
+    <div className="cf-container-wrapper">
```

Comment:
could this be done without wrapping it in another div container?

Reply:
Maybe we can discuss this as a parking lot in standup. This is what the `SingleColumn.tsx` component does so I followed in that same vein. 

Off the top of my head I can't think of anything but maybe we can come up with an alternate solution during the parking lot

Reply:
Addressed via your idea of only having a wrapper div when it's empty